### PR TITLE
Zs 232 enable reporting

### PR DESCRIPTION
**Pull request by smvdheijden**
[@MicroGrids]

### CODE REVIEW

File: cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/glue/steps/database/adapterprotocolie61850/Iec61850DeviceSteps.java [hunk 1]
```diff
@@ -86,11 +89,14 @@ private void createIec61850Device(final Map<String, String> settings) {
...
         final Iec61850Device iec61850Device = new Iec61850Device(
-                getString(settings, PlatformMicrogridsKeys.KEY_DEVICE_IDENTIFICATION, PlatformMicrogridsDefaults.DEFAULT_DEVICE_IDENTIFICATION));
+                getString(settings, PlatformMicrogridsKeys.KEY_DEVICE_IDENTIFICATION,
+                        PlatformMicrogridsDefaults.DEFAULT_DEVICE_IDENTIFICATION));
 
         iec61850Device.setIcdFilename(getString(settings, PlatformMicrogridsKeys.KEY_IEC61850_ICD_FILENAME));
         iec61850Device.setPort(getInteger(settings, PlatformMicrogridsKeys.KEY_IEC61850_PORT));
         iec61850Device.setServerName(getString(settings, PlatformMicrogridsKeys.KEY_IEC61850_SERVERNAME));
+        iec61850Device.setEnableAllReportsOnConnect(Helpers.getBoolean(settings,
+                PlatformMicrogridsKeys.ENABLE_ALL_REPORTS, PlatformMicrogridsDefaults.ENABLE_ALL_REPORTS));
```
Comment (renevanr) [inline cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/glue/steps/database/adapterprotocolie61850/Iec61850DeviceSteps.java]: import static Helpers.getBoolean (like getString)

File: cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/PlatformMicrogridsDefaults.java [hunk 2]
```diff
@@ -11,4 +11,6 @@
 
 public class PlatformMicrogridsDefaults extends com.alliander.osgp.cucumber.platform.PlatformDefaults {
 
+    public static final Boolean ENABLE_ALL_REPORTS = true;
```
Comment (renevanr) [inline cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/PlatformMicrogridsDefaults.java]: Default false seems more logical, but difficult for me to know the impact

Comment (smvdheijden) [reply cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/PlatformMicrogridsDefaults.java]: I think it's more logical to have a default value set to true as this is only for the tests for microgrids, where reporting should always be enabled.

File: cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/glue/steps/database/adapterprotocolie61850/Iec61850DeviceSteps.java (hunk 1 above)
Comment (smvdheijden) [reply cucumber-tests-platform-microgrids/src/test/java/com/alliander/osgp/cucumber/platform/microgrids/glue/steps/database/adapterprotocolie61850/Iec61850DeviceSteps.java]: Done

**Pull request merged**